atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampUtils.java

```diff
@@ -93,20 +94,29 @@ public static CqlQuery constructCheckAndSetMultipleQuery(Map<String, Pair<byte[]
             builder.append(constructCheckAndSetQuery(columnName, expected, target));
         });
         builder.append("APPLY BATCH;");
-        return new CqlQuery(builder.toString());
+
+        // This looks awkward. However, we know that all expressions in this String pertain to timestamps and known
```

Comment:
OK, as long as we actually fix this soon. OK with this to get out of p0 situation

Reply:
This is addressed in #3472